packages/service-library/src/servicelib/utils.py

```diff
@@ -69,6 +69,8 @@ def fire_and_forget_task(
     def log_exception_callback(fut: asyncio.Future):
         try:
             fut.result()
+        except asyncio.CancelledError:
+            logger.exception("Task was cancelled")
```

Comment:
IMO the log message is not very informative provided that ``CancelledError`` is descriptive enough. 

Also, a ``Task`` is a future-like object but not exactly the same, right?

what about logging 
```python
 logger.exception (f"{fut} spawned as fire&forget was cancelled") 
```

Reply:
This is acceptable.